src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/ProjectsController.kt

```diff
@@ -86,6 +89,27 @@ class ProjectsController(private val session: Session,
         return SuccessResponse(project).asResponseEntity()
     }
 
+    @PostMapping("/project/{projectId}/note")
+    @ResponseBody
+    fun saveProjectNote(
+            @PathVariable("projectId") projectId: Int,
+            @RequestParam("note") note: String): ResponseEntity<String>
+    {
+        projectRepository.saveProjectNote(projectId, note, userId())
+        return EmptySuccessResponse.asResponseEntity()
+    }
+
+    @PostMapping("project/{projectId}/version/{versionId}/note")
```

Comment:
```suggestion
    @PostMapping("/project/{projectId}/version/{versionId}/note")
```
I know there's already a bit of inconsistency here, but routes should start with `/`.